src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -18,7 +18,12 @@
 from azure.cli.core._util import CLIError
 
 logger = _logging.get_az_logger(__name__)
-BASE_URL = "https://westus.mindaro.microsoft.io"
+
+try:
+    BASE_URL = os.environ['AZURE_CLI_CONTAINER_SERVICE_URL']
+except KeyError:
+    BASE_URL = "https://westus.mindaro.microsoft.io"
```

Comment:
We have config support.

```
from azure.cli.core._config import az_config
BASE_URL = az_config.get('container', 'service_url', fallback='https://westus.mindaro.microsoft.io')
```

With this, the config can be set in a few ways:
az_config will check the `AZURE_CONTAINER_SERVICE_URL` environment variable.
If that's not set, it will look in the config files.

You can run `az configure`, to verify that it's picking up the setting.

Link to the implementation: https://github.com/Azure/azure-cli/blob/487f0196fc30715f5cf70b7c781185b5c2a5bf18/src/azure-cli-core/azure/cli/core/_config.py#L42).

Examples of use:
https://github.com/Azure/azure-cli/blob/487f0196fc30715f5cf70b7c781185b5c2a5bf18/src/azure-cli-core/azure/cli/core/application.py#L190

https://github.com/Azure/azure-cli/blob/487f0196fc30715f5cf70b7c781185b5c2a5bf18/src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_validators.py#L49